index.d.ts

```diff
@@ -0,0 +1,420 @@
+// Type definitions for express-validator 3.0.0
+// Project: https://github.com/ctavan/express-validator
+// Definitions by: Ayman Nedjmeddine <https://github.com/IOAyman>, Nathan Ridley <https://github.com/axefrog/>, Jonathan Häberle <http://dreampulse.de>, Peter Harris <https://github.com/codeanimal/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference types="express"/>
+///<reference types="bluebird"/>
+
+// Add RequestValidation Interface on to Express's Request Interface.
+declare namespace Express {
+  interface Request extends ExpressValidator.RequestValidation {}
+}
+
+// External express-validator module.
+declare module "express-validator" {
+  import express = require('express');
+
+  /**
+   * @param options see: https://github.com/ctavan/express-validator#middleware-options
+   * @constructor
+   */
+  function ExpressValidator(options?: ExpressValidator.Options.ExpressValidatorOptions): express.RequestHandler;
+
+  export = ExpressValidator;
+}
+
+// Internal Module.
+declare namespace ExpressValidator {
+
+  interface ValidatorFunction { (item: string | {}, message?: string): Validator; }
```

Comment:
You mean an overload that accepts _'only the schema'_ object :) You're right. Will do.